src/main/java/core/basesyntax/services/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.FileWriterService;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void writeToFile(String fileName, String report) {
+        try {
+            Files.writeString(Paths.get(filePath), report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find file by path" + filePath);
```

Comment:
Consider including the original `IOException` as a cause when throwing the `RuntimeException` to provide more context for debugging. For example: `throw new RuntimeException("Can't find file by path " + fileName, e);`